docs/src/main/tut/docs/core/interpreters/README.md

```diff
@@ -24,12 +24,12 @@ import freestyle._
 import cats.implicits._
 
 @free trait KVStore {
-  def put[A](key: String, value: A): OpSeq[Unit]
-  def get[A](key: String): OpSeq[Option[A]]
-  def delete(key: String): OpSeq[Unit]
+  def put[A](key: String, value: A): FS[Unit]
+  def get[A](key: String): FS[Option[A]]
+  def delete(key: String): FS[Unit]
   def update[A](key: String, f: A => A): OpSeq[Unit] =
-    get[A](key) flatMap {
-      case Some(a) => put[A](key, f(a))
+    get[A](key).freeS flatMap {
```

Comment:
If we are keeping `OpSeq` and `OpPar` around for special cases like this one I think we should rename then to something like `FS.Seq` and `FS.Par` so they remain consistent with the naming conventions for return types in `@free` declarations